snmp/datadog_checks/snmp/data/default_profiles/hp-h3c-switch.yaml

```diff
@@ -0,0 +1,68 @@
+extends:
+  - _generic-if.yaml
+  - _hp.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.25506.11.1.*
+
+metrics:
+  - MIB: HH3C-ENTITY-EXT-MIB
+    table:
+      OID: 1.3.6.1.4.1.25506.2.6.1.4.1
+      name: hh3cProcessTable
+    symbols:
+      - OID: 1.3.6.1.4.1.25506.2.6.1.4.1.1.3
+        name: hh3cProcessUtil5Min
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.25506.2.6.1.4.1.1.1
+          name: hh3cProcessID
+        tag: hh3c_process_id
+      - column:
+          OID: 1.3.6.1.4.1.25506.2.6.1.4.1.1.2
+          name: hh3cProcessName
+        tag: hh3c_process_name
+  - MIB: HH3C-ENTITY-EXT-MIB
+    table:
+      OID: 1.3.6.1.4.1.25506.2.6.1.1.1
+      name: hh3cEntityExtStateTable
+    symbols:
+      - OID: 1.3.6.1.4.1.25506.2.6.1.1.1.1.6
+        name: cpu.usage
+    metric_tags:
+      - MIB: HH3C-ENTITY-EXT-MIB
+        column:
+          OID: 1.3.6.1.4.1.25506.2.6.1.1.1.1.1
+          name: hh3cEntityExtPhysicalIndex
+        tag: hh3c_entity_ext_physical_index
```

Comment:
```suggestion
```

🥜 Maybe we can remove `hh3c_entity_ext_physical_index` tag since the `cpu` tag has the same value.

Reply:
Same for following table.

Reply:
addressed in [06699b6](https://github.com/DataDog/integrations-core/pull/15207/commits/06699b6c57557a49d868eed6b4d245178eb9a5fa) (`hp-h3c-switch` profile is added in a separate PR, this branch is based of of the other one)